content/SCALETutorials/Network/Interfaces/SettingUpStaticIPs.md

```diff
@@ -9,7 +9,7 @@ tags:
 - DHCP
 ---
 
-This article describes setting up a network interface with a static IP address or changing the main interface from a DHCP-assigned to a manually-entered static IP address.
+This article describes setting up a network interface with a static IP address, also referred to as an alias, or changing the main interface from a DHCP-assigned to a manually-entered static IP address.
```

Comment:
a static IP is not always an alias, but it can be an alias so this statement is incorrect.
```suggestion
This article describes setting up a network interface with a static IP address or changing the main interface from a DHCP-assigned to a manually-entered static IP address.
```